tests/metrics_test.go

```diff
@@ -246,8 +196,9 @@ var _ = Describe("Metrics", func() {
 			unpauseSsp()
 			waitUntilDeployed()
 
-			newRestoredCount := totalRestoredTemplatesCount()
-			Expect(newRestoredCount).To(BeZero())
+			Consistently(func() int {
+				return totalRestoredTemplatesCount()
+			}, 5*time.Minute, 10*time.Second).Should(Equal(restoredCount))
```

Comment:
Are 5 minutes really necessary? Tests already take a long time to run, and we try to minimize the time.